src/frontend/components/UI/QuestsViewer/components/QuestDetailsWrapper/index.tsx

```diff
@@ -89,6 +93,63 @@ async function getRewardClaimGasEstimation(reward: Reward) {
   return gasNeeded
 }
 
+interface ConfirmProps extends ModalProps {
+  networkName: string
+  onConfirm: () => void
+  onCancel: () => void
+}
+
+function ConfirmClaimModal(props: ConfirmProps) {
+  const { t } = useTranslation()
+  return (
+    <Modal {...props} classNames={{ root: styles.confirmModal }}>
+      <Images.AlertTriangle
+        className={styles.alertIcon}
+        width={24}
+        height={24}
+      />
+      <div className={styles.confirmTextWrapper}>
+        <Modal.Body className={styles.confirmText}>
+          <Modal.Title>
+            {t('quest.claimWarning.title', 'Confirm Quest Reward Claim')}
+          </Modal.Title>
+          <span>
+            <span className="text--bold">
+              {t('quest.claimWarning.important', 'IMPORTANT')}:
+            </span>{' '}
+            {t(
+              'quest.claimWarning.networkMessage',
+              `Please ensure that you are allocating enough gas on the {{networkName}} network for the transaction to be successfully confirmed`,
+              { networkName: props.networkName }
+            )}{' '}
+            <span className="text--bold">
+              {t('quest.claimWarning.within24Hrs', 'within 24 hrs')}
```

Comment:
this will have issues translating. 'react-i18next' has a `Trans` component for this so that you can keep the translation string together https://stackoverflow.com/questions/75309102/how-to-add-a-partial-style-for-a-i18n-translation

since in other languages each component of the sentence is not always in the same order